site/docs/guides/components-basics.md

```diff
@@ -8,17 +8,17 @@ This guide walks you through the basics of how to style and work with components
 
 You use **Stylable** with a component file (for example using React), along with a **Stylable** CSS file that has the extention `.st.css`.
 
-> **Note**:
+> **Note**:  
 > This guide shows the JSX side of our [stylable-integration](https://github.com/wixplosives/stylable-integration) with React. 
 
-**Stylable** styles are similar to a type-system. Once you have declared that a CSS class is of the type `Button` for example, **Stylable** knows its internal structure and can match its internal parts and states.
+**Stylable** styles are similar to a type-system. Once you have declared that a CSS class is of the type `button` for example, **Stylable** knows its internal structure and can match its internal parts and states.
```

Comment:
Same here. capitalized.

Reply:
Here it says that the "class" is of type button. Shouldn't any "class" be camelCase?

Reply:
still a component.